kratos/python/add_properties_to_python.cpp

```diff
@@ -34,76 +34,96 @@ namespace Python
 {
 using namespace pybind11;
 
-typedef Mesh<Node<3>, Properties, Element, Condition> MeshType;
+typedef Node<3> NodeType;
+typedef Mesh<NodeType, Properties, Element, Condition> MeshType;
 typedef ConstitutiveLaw ConstitutiveLawBaseType;
-
+typedef std::size_t IndexType;
+typedef PointerVectorSet<Properties, IndexedObject> PropertiesContainerType;
 
 template< class TContainerType, class TVariableType >
-bool HasHelperFunction_Element(TContainerType& el, const TVariableType& rVar)
+bool HasHelperFunction_Element(TContainerType& rProperties, const TVariableType& rVar)
```

Comment:
name of funciton

Reply:
The name was already like that

Reply:
but still it does not fit here .. we can change it in this PR IMO.